docs/config.md

```diff
@@ -1455,41 +1455,49 @@ Setting this value to zero will disable the limit entirely.
 
 ## Ways to specify `jj` config: details
 
-### User config file
+### User config files
 
-An easy way to find the user config file is:
+An easy way to find the user config file/directory is:
 
 ```bash
 jj config path --user
 ```
 
-The rest of this section covers the details of where this file can be located.
+On all platforms, the user's global `jj` configurations are loaded in the following
+precedence order (with later configs overriding earlier ones):
 
-On all platforms, the user's global `jj` configuration file is located at either
-`~/.jjconfig.toml` (where `~` represents `$HOME` on Unix-likes, or
-`%USERPROFILE%` on Windows) or in a platform-specific directory. The
-platform-specific location is recommended for better integration with platform
-services. It is an error for both of these files to exist.
+- `$HOME/.jjconfig.toml` (not recommended)
+- `$XDG_CONFIG_HOME/jj/config.toml`
+- `$XDG_CONFIG_HOME/jj/conf.d/*.toml`
+
+where `$HOME` represents the user's home directory (`%USERPROFILE%` on Windows),
+and `$XDG_CONFIG_HOME` represents the platform-specific configuration directory.
+The platform-specific location is recommended for better integration with platform
+services.
+
+The `conf.d` directory loads additional files in lexicographic order. This allows
```

Comment:
nit: maybe you mean "Files in the `conf.d` directory are loaded in lexicographical order."?